desktop/cmp/tab/container/TabContainerModel.js

```diff
@@ -85,14 +85,14 @@ export class TabContainerModel {
 
         throwIf(!tab, `Unknown Tab ${id} in TabContainer.`);
 
-        this.activeTabId = id;
-        if (tab.reloadOnShow) tab.requestRefresh();
-
         if (route && !suppressRouting) {
             const tabRoute = route + '.' + id;
             if (!XH.router.isActive(tabRoute)) {
                 XH.navigate(tabRoute);
             }
```

Comment:
John -- I can see wanting to return early and not set the activeTabId if we are going to navigate to it.  

But with the code as is, aren't we risking missing setting the id (and doing a needed refresh) if the route is already correct (e.g. conditional on line 90 returns false)

Does your code correctly do the reloadOnShow in all cases?